paperscraper/lib.py

```diff
@@ -198,13 +198,20 @@ async def pubmed_to_pdf(pubmed_id, path, session: ClientSession) -> None:
     await pmc_to_pdf(pmc_id, path, session)
 
 
-async def pmc_to_pdf(pmc_id, path, session: ClientSession) -> None:
+async def pmc_to_pdf(
+    pmc_id: str, path: str | os.PathLike, session: ClientSession
+) -> None:
     pdf_url = await find_pmc_pdf_link(pmc_id, session)
     async with session.get(pdf_url, allow_redirects=True) as r:
-        if not r.ok or not await likely_pdf(r):
+        exc: Exception | None = None
+        with contextlib.suppress(ClientResponseError):
```

Comment:
There was a `ruff` mix up here with SIM105, fixed in https://github.com/blackadad/paper-scraper/pull/72